packages/create-platformatic/src/say.mjs

```diff
@@ -0,0 +1,28 @@
+import readline from 'node:readline'
+import logUpdate from 'log-update'
+import { pltGreen } from './colors.mjs'
+import { sleep, randomBetween } from './utils.mjs'
+
+export const say = async (messages, { clear = false } = {}) => {
+  const rl = readline.createInterface({ input: process.stdin, escapeCodeTimeout: 50 })
```

Comment:
why are we creating a new readline every time this function is called?

Reply:
It's a leftover of a different implementation, tnx for spotting this.